fuzzers/baby/backtrace_baby_fuzzers/c_code_with_fork_executor/src/main.rs

```diff
@@ -47,7 +47,7 @@ pub fn main() {
     };
     // Create an observation channel using the signals map
     let observer =
-        unsafe { ConstMapObserver::<u8, 3>::from_mut_ptr("signals", ptr::NonNull::new(map_ptr)) };
+        unsafe { ConstMapObserver::<u8, 3>::from_mut_ptr("signals", ptr::NonNull::new(map_ptr).expect("map ptr is null.") ) };
```

Comment:
can you use `nonnull!()`?

Reply:
Ah wait it's not a constant, nevermind

Reply:
yup

Reply:
Wait, how can it be not constant for const map observer? :D

Reply:
what's constant is the size, not the ptr.
it is still possible to have constant maps with buffers on the heap.
i guess it's what we want, right?